### ISSUE TITLE
Data shows IRI and Road length (ORMA) but doesn't show geometry

### ISSUE THREAD

**Issue by PCDuc** [orma/openroads-vn-analytics#603]
This is the case of VPRoMMS ID 232CL00008 which has been reported as fail to upload in ticket #545 
IRI and Length (ORMA) value is showed but has no geometry. The stat also shows no data has been collected.

![image](https://user-images.githubusercontent.com/31137016/53523641-eb50cc80-3b0f-11e9-907a-7132e2f6a978.png)

![image](https://user-images.githubusercontent.com/31137016/53523643-edb32680-3b0f-11e9-8ad9-4022a0c00d64.png)

What action should be done to resolve this situation? Does it need to remove all VPRoMMS ID which failed to upload and re-upload those?

**Comment by geohacker** [comment]
@PCDuc you're right. In this case, you'd have to remove the VPROMM data from ORMA and re-upload.
